orca-webhook/src/main/groovy/com/netflix/spinnaker/orca/webhook/tasks/CreateWebhookTask.groovy

```diff
@@ -162,4 +157,41 @@ class CreateWebhookTask implements RetryableTask {
       return TaskResult.builder(ExecutionStatus.TERMINAL).context(outputs).build()
     }
   }
+
```

Comment:
Below method adjusts the status check URL if the original webhook domain and status check URL domain given back matches.